diff -Naur ccminer-2.3-tpruvot.old/Makefile.am ccminer-2.3-tpruvot/Makefile.am
--- ccminer-2.3-tpruvot.old/Makefile.am	2019-01-03 11:10:55.809208810 +0100
+++ ccminer-2.3-tpruvot/Makefile.am	2019-01-03 11:12:13.305606565 +0100
@@ -116,7 +116,9 @@
 ccminer_LDADD += -lcuda
 
 nvcc_ARCH :=
-#nvcc_ARCH += -gencode=arch=compute_61,code=\"sm_61,compute_61\"
+nvcc_ARCH += -gencode=arch=compute_75,code=\"sm_75,compute_75\"
+nvcc_ARCH += -gencode=arch=compute_70,code=\"sm_70,compute_70\"
+nvcc_ARCH += -gencode=arch=compute_61,code=\"sm_61,compute_61\"
 nvcc_ARCH += -gencode=arch=compute_52,code=\"sm_52,compute_52\"
 nvcc_ARCH += -gencode=arch=compute_50,code=\"sm_50,compute_50\"
 #nvcc_ARCH += -gencode=arch=compute_35,code=\"sm_35,compute_35\"
